app/AccountCreation.tsx

```diff
@@ -31,7 +31,7 @@ export function AccountCreation(props: {
             type="text"
             placeholder="enter your username"
             value={username}
-            onChange={(e) => {
+            onChange={(e:any) => {
```

Comment:
any reason this is now required when previously it wasn't? also we're never allowed to use `any`

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/10/commits/af12b81a211058fa794bea8a0be303d6b06a1b60 
I don't know why it flagged last time, but it's been removed and it's not complaining